appuio-specifics-public.rst

```diff
@@ -22,20 +22,26 @@ URLs and Domains
 - Logging URL: https://logging.appuio.ch/
 - Application Domain: ``appuioapp.ch``
 
+.. _persistent-storage:
+
 Persistent Storage
 ------------------
 
-APPUiO currently uses GlusterFS based persistent storage.
-In case of databases we provide gluster volumes with storage class
-``gluster-database`` to avoid instability and make use of `optimized parameters <https://github.com/gluster/glusterfs/blob/release-7/extras/group-db-workload>`__.
-For now volumes with the following sizes are available out of the box:
+APPUiO currently uses GlusterFS based persistent storage. For database data
+we provide Gluster volumes with storage class ``gluster-database``
+to avoid :ref:`instability <faq-database>`, which makes use of
+`optimized parameters <https://github.com/gluster/glusterfs/blob/release-7/extras/group-db-workload>`__.
+(Please set the ``storageClassName`` attribute in your `volume claim
+<https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims>`__
+or `StatefulSet <https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#components>`__
+manifest accordingly.) For now, volumes with the following sizes are available out of the box:
 
 * 1 GiB
 * 5 GiB
 
-However you can contact us to get larger volumes: `Contact <http://appuio.ch/#contact>`__.
+If you need larger volumes please `contact us <https://appuio.ch/#contact>`__.
```

Comment:
Should we use the email address(es) throughout the documentation for customer contacts, or this link to the contact form? – Who are the recipients of hello@appuio.ch and support@appuio.ch, by the way?

Reply:
I would suggest to refer to support@appuio.ch or even better to https://control.vshn.net (APPUiO users have access). support@appuio.ch goes to our Jira and hello@appuio.ch to many Puzzlers and info@vshn.ch.